src/game.cpp

```diff
@@ -11442,6 +11442,22 @@ void game::unload( item &it )
         return;
     }
 
+    // Handle magazines as a special case
+    if( target->is_magazine() ) {
+        // Remove all contained ammo consuming half as much time as required to load the magazine
+        target->contents.erase( std::remove_if( target->contents.begin(), target->contents.end(), [&]( item& e ) {
```

Comment:
Not to cancel but because `add_or_drop_with_msg` prompts for empty containers where necessary and could fail if earlier unloaded ammo has consumed them all.

It's a trivial detail but one I'd like to get correct now if at all possible.